tests/pet.test.js

```diff
@@ -0,0 +1,112 @@
+const Pet = require('../src/pet');
+
+describe('create a new object with a set of properties', () => {
+  let pet;
+  beforeEach(() => {
+    pet = new Pet('Boris');
+  });
+
+    test('returns an object', () => {
+      expect(new Pet('Boris')).toBeInstanceOf(Object);
```

Comment:
you have already made a pet called boris in your before each, so you can use that here instead of creating another one in your expect